hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/handlers/ContractUpdateHandler.java

```diff
@@ -38,6 +54,10 @@
  */
 @Singleton
 public class ContractUpdateHandler implements TransactionHandler {
+
+    private static final Key IMMUTABILITY_SENTINEL_KEY =
+            Key.newBuilder().keyList(KeyList.newBuilder()).build();
+
```

Comment:
We could probably add this variable in SPI, so we can use in both Token Service and contract service 😃 . I see it is defined in multiple places today.

Reply:
I see there is a `HapiUtils.EMPTY_KEY_LIST` public constant. I will use that 👍